addons/api/addon/serializers/connection-recording.js

```diff
@@ -12,4 +12,16 @@ export default class ConnectionSerializer extends ApplicationSerializer.extend(
   attrs = {
     channel_recordings: { embedded: 'always' },
   };
+
+  normalize(typeClass, hash, ...rest) {
+    const normalizedHash = structuredClone(hash);
+    const normalized = super.normalize(typeClass, normalizedHash, ...rest);
+    // we should not override the ember models errors field,
+    // so we rename the errors field from the api
+
+    const err = normalized.data.attributes.errors_session_recording;
+    delete normalized.data.attributes.errors_session_recording;
+    normalized.data.attributes.errors = err;
+    return normalized;
```

Comment:
You have this mixed up, we are receiving it as `error` and we're renaming it to the new variable.